src/AkismetSpamProtector.php

```diff
@@ -56,37 +64,26 @@ class AkismetSpamProtector implements SpamProtector
      * @var array
      */
     private $fieldMapping = array();
-    
-    /**
-     * Overridden API key
-     *
-     * @var string
-     */
-    protected static $_api_key = null;
```

Comment:
Can you add this back, just without the leading underscore?

Reply:
It's defined as a private at the beginning of the class: https://github.com/creative-commoners/silverstripe-akismet/blob/ec0c41d598817e216a7a7c0467c7b3dbce5a0578/src/AkismetSpamProtector.php#L19-L25

Reply:
Ah, true :)

Reply:
Ok, this property has a different purpose by the looks of it, so we'll need to retain it.

Reply:
Following up, I see this is done on a different line as `protected $apiKey = '';`, which is cool :)